### ISSUE TITLE
Add ability to remove -G option from command to get raw output

### ISSUE THREAD

**Issue by justinc11** [Pisamad/p4api#6]
I would like to get a unified diff between two versions of a file, and the `-G` option prevents `p4 diff2 -u //some/file#1 //some/file#2` from returning appropriate results.

It looks like [this repo](https://github.com/narthur157/p4-local-review) has worked around this by using the p4-oo package, but would be nice to have everything available in p4api.

**Comment by Pisamad** [comment]
Hi justinc11,
The -G option is the main feature of p4api, but I can understand the need to have a raw level for some P4 commands.
I guess I could just add rawCmd and rawCmdSync methods to the p4api class.
For now, I do not have much time for p4api, but I will try to program a new version for at least September 15th.
Regards

**Comment by Pisamad** [comment]
Hello Justinc11,
You will find version V3.3.0 on the "raw" branch that implements the new rawCmd() & rawCmdSync() methods.
Tell me if this version meets your needs, then I will publish it.

**Comment by justinc11** [comment]
Thanks @Pisamad. It works well from what I have tried so far.

The only suggestion I have would be to change the keys of the response to something like `error` and `text` to be more consistent with the other command functions.

So [line 286](/Pisamad/p4api/blob/raw/src/index.js#L286) and [line 324](/Pisamad/p4api/blob/raw/src/index.js#L324) would become something like this:

```javascript
 const result = {
    text: dataOut.toString(),
    error: dataErr.toString()
 }
```

**Comment by Pisamad** [comment]
Pertinent suggestion. I'm going to do that, then publish to npm.

**Comment by Pisamad** [comment]
Solved in V3.3.0
